diff --git a/MODULE.bazel b/MODULE.bazel
index 71ae8e7b6..05e4ac959 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -74,7 +75,10 @@ use_repo(
 
 maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
 
+MAVEN_REPO = "maven"
+
 maven.install(
+    name = MAVEN_REPO,
     artifacts = IO_GRPC_GRPC_JAVA_ARTIFACTS,
     repositories = [
         "https://repo.maven.apache.org/maven2/",
@@ -82,7 +86,7 @@ maven.install(
     strict_visibility = True,
 )
 
-use_repo(maven, "maven")
+use_repo(maven, maven = MAVEN_REPO)
 
 maven.override(
     coordinates = "com.google.protobuf:protobuf-java",
